info/machine.go

```diff
@@ -93,6 +93,9 @@ type DiskInfo struct {
 
 	// Size in bytes
 	Size uint64 `json:"size"`
+
+	// Scheduler - one of "none", "noop", "cfq", "deadline"
```

Comment:
just the comment, or field name, or json tag? I am assuming its just the comment :)